.ci/setup_env_fedora.sh

```diff
@@ -50,6 +50,9 @@ chronic sudo -E dnf -y install btrfs-progs-devel device-mapper-devel 	  \
 	libgpg-error-devel libseccomp-devel libselinux-devel ostree-devel \
 	pkgconfig go-md2man
 
+echo "Install cri-containerd dependencies"
+chronic sudo -E dnf -y install libseccomp-devel btrfs-progs-devel libseccomp-static
```

Comment:
OOI, why do we need the `-static` version of this library? It doesn't appear to be needed on Ubuntu too.